packages/docs/docs/docs/linking.md

```diff
@@ -1,127 +1,281 @@
+
+To link the OpenZeppelin contracts EVM package into your project, simply run the following:
+
+```console
+zos link openzeppelin-eth@2.2.0
+```
+
+This command will download the EVM package (bundled as a regular npm package), and connect it to your ZeppelinOS project. We now have all of OpenZeppelin contracts at our disposal, so let's create an ERC20 token!
+
+> Note: Make sure you install `openzeppelin-eth` and not `openzeppelin-solidity`. The latter is set up for general usage, while `openzeppelin-eth` is tailored for being used with ZeppelinOS. This means that its contracts are [already set up to be upgradeable](writing_contracts#use-upgradeable-packages).
+
+## Creating an ERC20 token
+
+Let's deploy an ERC20 token contract to our development network. But first, make sure to install ganache if needed, and start a new instance:
+
+```console
+npm install --global ganache-cli
+ganache-cli --deterministic
+```
+
+For setting up the token, we will be using the [StandaloneERC20 implementation](https://github.com/OpenZeppelin/openzeppelin-eth/blob/master/contracts/token/ERC20/StandaloneERC20.sol) provided by the OpenZeppelin package. We will _initialize_ the instance with the token metadata (name, symbol, and decimals), and mint a large initial supply for one of our accounts.
+
+<!-- CODE: We need a command to retrieve the current accounts (#967) -->
+
+> Note: Your available accounts are shown by ganache when you start the process. If you ran it with the `--deterministic` flag as instructed, then your first and default account will be `0x90f8bf6a479f320ead074411a4b0e7944ea8c9c1`.
+
+```console
+$ zos create
+? Pick a contract to instantiate: openzeppelin-eth/StandaloneERC20
+? Pick a network: development
+✓ Deploying openzeppelin-eth dependency to network
+? Do you want to call a function on the instance after creating it?: Yes
+? Select which function: * initialize(name: string, symbol: string, decimals: uint8, initialSupply: uint256, initialHolder: address, minters: address[], pausers: address[])
+? name (string): MyToken
+? symbol (string): MYT
+? decimals (uint8): 18
+? initialSupply (uint256): 100e18
+? initialHolder (address): 0x90f8bf6a479f320ead074411a4b0e7944ea8c9c1
+? minters (address[]): 
+? pausers (address[]): 
+✓ Setting everything up to create contract instances
+✓ Instance created at 0x2612Af3A521c2df9EAF28422Ca335b04AdF3ac66
+```
+
+Let's break down what we did in the command above. We first chose to create an instance of the `StandaloneERC20` contract from the `openzeppelin-eth` package we had linked before, and to create it in the local `development` network. We are then instructing the CLI to _initialize_ it with the initial values needed to set up our token. This requires us to choose the appropriate `initialize` function, and input all the required arguments. ZeppelinOS will then atomically create and initialize the new instance in a single transaction.
+
```

Comment:
Zos use initialize rather than constructor, and this is one of the core concepts.

We can take reader attention here by saying something like this.

"Look we are using initializer, rather than constructor, here is a link to read more about initialize function."

This will create a proper distinction in readers mind that in zos we use initialize, not constructor.

Because when the user read the text above, she might not create a proper distinction b/s constructor and initialize.

Reply:
This is covered later in L93. I wanted to only touch the concept of creating a contract from an EVM package here, so I left initializers for later, when we are deploying a contract just link in `first.md`.